src/components/Refresh/Refresh.css

```diff
@@ -0,0 +1,10 @@
+@import "../../index.css";
+
+.RefreshButton {
+  padding: 12px 0 0;
+  margin-left: auto;
+  margin-right: auto;
+  display: block;
+  margin-top: 20px;
+}
```

Comment:
Can we stick to default material-ui styles, or if not, can we make this reusable (like [this](https://github.com/bayesimpact/tds-frontend/blob/04dfe5d63d1c24e3bf4f23c13ec03cb61e8ea9ea/src/components/App/App.css#L71))?